content/en/docs/how-to/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: How-to guides
-description: The collection of guides for configuring and using Notary
+description: The collection of guides for configuring and using The Notary Project
```

Comment:
If all the How-to guides are related to Notation, I would suggestion `using Notation`

Reply:
holding off until #312 is resolved